### DISCUSSION TITLE
Getting default enums from symbolTable?

### DISCUSSION THREAD

**Discussion by bryanreif** [microsoft/Power-Fx#2234] [Q&A]
If I try to look up the default enums in the symbolTable, I get false. Is this intended? TIA

```
var config = new PowerFxConfig(Features.PowerFxV1);
var symbolTable = config.SymbolTable;
symbolTable.TryLookupSlot(LanguageConstants.ColorEnumString, out var slot);
```

**Comment by anderson-joyle** [comment]
Slots will store elements added by the user only, like variables and/or formulas. Any default symbols like functions or enums will be stored somewhere else.

**Comment by MikeStall** [comment]
We're getting better docks here, but there's also some more written here:
https://github.com/microsoft/Power-Fx/discussions/987
